nixos/modules/security/systemd-confinement.nix

```diff
@@ -194,6 +195,9 @@ in {
           echo "BindReadOnlyPaths=$storePath"
         fi
       done < "$closureInfo/store-paths" >> "$serviceFile"
+
+      # add i18n locale as well ($LOCAL_ARCHIVE)
+      echo "BindReadOnlyPaths=$i18nGlibcLocales" >> "$serviceFile"
```

Comment:
Sure, can do. Note that if you have anything linked to glibc you pull in pkgs.glibcLocales already by default so this really isn't adding anything to the chroot (just you now have two versions instead of one); but I can understand not having locales at all.

FWIW all my confinements are binSh = null and mode = "chroot-only" and I now have quite a few :/ Some services like postgres need /proc/self/fd or /dev/null but I'd rather just mount the required directory rather than the whole thing. `fullUnit` is that same that kind of makes confinement a bit moot if you expose all of this, ProtectSystem is the just about the same... (btw see #87420 about ProtectSystem, forgot to cc you)

Anyway, updating with default package set.